libutils/string_lib.c

```diff
@@ -803,26 +803,26 @@ bool ReplaceStr(const char *in, char *out, int outSz, const char *from, const ch
 /**
  * Replace all occurrences of #find with #replace.
  *
```

Comment:
Extra comma in the commit message's title.

Reply:
Fixed